react-components/src/components/RevealToolbar/LayersButton/updateExternalStateFromLayerHandlers.test.ts

```diff
@@ -0,0 +1,69 @@
+import { beforeEach, describe, expect, type Mock, test, vi } from 'vitest';
+import { updateExternalStateFromLayerHandlers } from './updateExternalStateFromLayerHandlers';
+import {
+  createCadHandlerMock,
+  createImage360HandlerMock,
+  createPointCloudHandlerMock
+} from '../../../../tests/tests-utilities/fixtures/modelHandler';
+import { type LayersUrlStateParam } from './types';
+import { type Dispatch, type SetStateAction } from 'react';
+
+describe(updateExternalStateFromLayerHandlers.name, () => {
+  let setLayerStateMock: Mock<Dispatch<SetStateAction<LayersUrlStateParam | undefined>>>;
+
+  beforeEach(() => {
+    setLayerStateMock = vi.fn<Dispatch<SetStateAction<LayersUrlStateParam | undefined>>>();
+  });
```

Comment:
Can't we move this mock to the test itself?

Reply:
:+1: